src/modules/battery.cpp

```diff
@@ -91,16 +67,16 @@ namespace modules {
         unsigned long current{std::strtoul(file_util::contents(m_frate).c_str(), nullptr, 10)};
         unsigned long voltage{std::strtoul(file_util::contents(m_fvoltage).c_str(), nullptr, 10)};
 
-        consumption = ((voltage / 1000.0) * (current /  1000.0)) / 1e6;
-      // if it was power, just use as is
+        consumption = ((voltage / 1000.0) * (current / 1000.0)) / 1e6;
+        // if it was power, just use as is
       } else {
         unsigned long power{std::strtoul(file_util::contents(m_frate).c_str(), nullptr, 10)};
 
         consumption = power / 1e6;
       }
 
       // convert to string with 2 decimmal places
-      string rtn(16, '\0'); // 16 should be plenty big. Cant see it needing more than 6/7..
+      string rtn(16, '\0');  // 16 should be plenty big. Cant see it needing more than 6/7..
```

Comment:
Please do not introduce indentation changes if it does not fix the issue. Code cleanup and refactoring is a good thing, but if you want your branch to be merged, you need to keep it clean and create another branch and PR for code cleanup.

Reply:
This is also fine, see my other comment :)